index.html

```diff
@@ -0,0 +1,329 @@
+      </div>
+    </section>
+  </main>
+  <section class="contact-me">
+    <div class="head-section-co">
+      <div id="head-container-co">
+        <h2 class="head-co">
+          About <br />
+          Myself
+        </h2>
+        <p class="description-co">
+          Hello I’m a software developer! I can help you build a product,
+          feature or website Look through some of my work and experience! If
+          you like what you see and have a project you need coded, don’t
+          hestiate to contact me.
+        </p>
+        <h2 class="connect-co">LET’S CONNECT</h2>
+
+        <ul class="social-media-icons">
+          <li>
+            <a href="#" target="_blank">
+              <img src="images/github.png" alt="git" />
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <img src="images/Linkedin.png" alt="linkedIn" />
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <img src="images/angelite.png" alt="image" />
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <img src="images/twitter.png" alt="image" />
+            </a>
+          </li>
+          <li>
+            <a href="#" target="_blank">
+              <img src="images/medium.png" alt="image" />
+            </a>
+          </li>
+        </ul>
+        <div class="new-code">
+          <button type="submit" class="go-newmes">My resume</button>
+        </div>
+
+        <div class="skill">
+          <div class="event12">
+            <div class="event7">
+              <h3 class="languages">Languages</h3>
+              <button type="submit" class="dropdown">
+                <img src="./images/Icon-Chevron-Down-Larger.png" alt="" />
+              </button>
+            </div>
+            <ul class="language-list">
+              <li class="event9">
+                <div class="lang">
+                  <img src="./images/Ellipse-1.png" alt="" />
+                  <p>JavaSript</p>
+                </div>
+              </li>
+              <li class="event10">
+                <div class="lang">
+                  <img src="./images/Ellipse-2.png" alt="" />
+                  <p>HTML</p>
+                </div>
+              </li>
+              <li class="event11">
+                <div class="lang">
+                  <img src="./images/Ellipse-3.png" alt="" />
+                  <p>CSS</p>
+                </div>
+              </li>
+            </ul>
+          </div>
+          <div class="event-skill">
+            <div class="event18">
+              <h3>Frameworks</h3>
+              <button type="submit" class="dropdown">
+                <img src="./images/Disabled.png" alt="" />
+              </button>
+            </div>
+            <img src="./images/Separator-Bottom.png" alt="" class="separator" />
+            <div class="event18">
+              <h3>Skills</h3>
```

Comment:
- [ ] kindly consider adding:

- the technology categories ("Languages", "Frameworks" and "Skills") in a ul, each of them in a li

- The technology in each category(e.g., "JavasScript", "HTML" and "CSS") in a list ul and each of them a li

  for semantic reasons  and remember it gives meaning to the html nodes, helping screen readers, web crawlers, etc

Reply:
Done